src/dinifile.d

```diff
@@ -103,12 +113,14 @@ extern (C++) const(char)* readFromEnv(StringTable* environment, const(char)* nam
 /*********************************
  * Write to our copy of the environment, not the real environment
  */
-extern (C++) static void writeToEnv(StringTable* environment, char* nameEqValue)
+extern (C++) static int writeToEnv(StringTable* environment, char* nameEqValue)
```

Comment:
1. Lose the `extern(C++)`
2. It's private, not static
3. The return value is set and used like it was a bool. Make it a bool.